_posts/2021-02-21-goodbye-scrum.md

```diff
@@ -0,0 +1,79 @@
+
+Additionally, I know that we enjoyed mobbing. It was invigorating and engaging, and, importantly, it helps minimize knowledge silos. Everyone seemed energized by the proposal that as our tasks wind down, we ramp up mobbing. However, mobbing has largely dropped by the wayside, and I don't think anything besides overcoming inertia has prevented us from mobbing more.
+
+One of our company's strategic priorities is "Unleash our people." And I would not be doing my job if I weren't constantly thinking about how to help you all be your best (professional) selves. Long story short, based on conversations I've had with [my peer] [Stephanie](https://www.linkedin.com/in/stephanie-simons-she-her-92444ab2/), who has a bunch of experience in using Kanban with her teams, I see it as a potential pathway to addressing these two issues.
+
+To give you some background, Kanban is not a software development process. In fact, as you may know, it originated in manufacturing, and rose to prevalence at Toyota on their production line under the direction of Taiichi Ohno in the 60s. It was brought to the broader business management community in the mid-80s by way of Eliyahu Goldratt's book, [The Goal](https://www.tocinstitute.org/the-goal-summary.html). In his book, Goldratt developed the Theory of Constraints which is derived from mathematical queueing theory and is one of the foundations of Kanban. It wasn't until 2010 that David Anderson documented his experience applying Kanban to software in his book [Kanban: Successful Evolutionary Change for Your Technology Business](https://shop.leankanban.com/products/kanban-successful-evolutionary-change-for-your-technology-business).
+
+Kanban, itself, is a meta-process designed to help teams implement continuous improvements in their production processes. Kanban has two rules:
+
+1. Make the work visible
+2. Limit work in progress (WIP)
+
+These two rules form the basis of implementing a Kanban system: a pull-based flow system, which, in turn, allows teams to continually optimize how work flows through the system.
+
+Making the work visible, I believe, will help address the first issue of projects feeling like they're stalling. In the absence of a (Jira) ticket, or one that is in flight for weeks, it is hard to visualize progress or detect when we've uncovered hidden complexity. If we are continually visualizing the work and seeing it flow through the system, I hypothesize it'll help alleviate the feeling that projects feel stalled. And, moreover, if we do detect that tasks are legitimately stalled because of incidental complexity, it'll be readily visible and something we can proactively address, therein contributing to forward progression and momentum. It's a virtuous cycle!
```

Comment:
`it is` -> `it's`

Reply:
`we are` -> `we're`

Reply:
This sentence is a bit clunky. Rewording it would help: `I believe, will help address the first issue of projects feeling like they’re stalling. In the absence of a (Jira) ticket, or one that is in flight for weeks, it is hard to visualize progress or detect when we’ve uncovered hidden complexity.`